src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/commonmodels/PersonName.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+public class PersonName {
+  @JsonProperty("forename")
+  private String forename;
+  @JsonProperty("surname")
+  private String surname;
+
+  public PersonName(String forename, String surname) {
+    setForename(forename);
+    setSurname(surname);
+  }
+
+  public String getForename() {
+    return forename;
+  }
+
+  public void setForename(String forename) {
+    if (forename == null) {
+      forename = "";
```

Comment:
Is there a reason why we set these to empty?

Reply:
Sorry this was a remnant from a point when I thought I would have to turn PersonName back into a String 

As null concatenated with a String is "null" I wanted to prevent someone from having the forename or surname null.

As this is no longer an issue it can be removed